config/jobs/kubernetes-sigs/azuredisk-csi-driver/azuredisk-csi-driver-config.yaml

```diff
@@ -631,7 +631,7 @@ presubmits:
             - name: NODE_MACHINE_TYPE
               value: "Standard_D4s_v3"
             - name: KUBERNETES_VERSION
-              value: "v1.23.5"
+              value: "latest-1.23"
```

Comment:
@andyzhangx is it intentional that we're running these tests on 1.23? Would it actually be better to run against "latest"?

Reply:
@andyzhangx gentle ping

Reply:
it's intentional since we want to run the e2e tests on different k8s version to get more k8s version coverage